test/dlc_tests/container_tests/bin/benchmark/get_mxnet_avg_throughput.py

```diff
@@ -0,0 +1,31 @@
+# usage: python <script_name>.py <log_file> 4 Training & Inference GPU
+# usage: python <script_name>.py <log_file> 1 Inference CPU
+import argparse
+parser = argparse.ArgumentParser()
```

Comment:
nit: The entire section of defining `parser` and `args` can go in the main-block. Also, please add two spaces between the first line of code and the last import statement.

(There is no need to address this now.

Reply:
Will address it in another PR. Thanks